libs/sdk-ui-charts/src/charts/headline/internal/HeadlinePagination.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2021 GoodData Corporation
+import React, { useState } from "react";
+
+import { HeadlinePaginationRenderer } from "./HeadlingePaginationRenderer";
+
+interface IHeadlinePaginationProps {
+    renderSecondaryItem?: (item: boolean) => JSX.Element;
+    renderTertiaryItem?: (item: boolean) => JSX.Element;
+}
+
+export const HeadlinePagination: React.FC<IHeadlinePaginationProps> = ({
+    renderSecondaryItem,
+    renderTertiaryItem,
+}) => {
+    const [item, setItem] = useState<number>(1);
+
+    const showNextItem = (): void => setItem(item + 1);
+
+    const showPrevItem = (): void => setItem(item - 1);
+
+    return (
+        <>
+            <HeadlinePaginationRenderer item={item} showNextItem={showNextItem} showPrevItem={showPrevItem} />
+            {renderSecondaryItem(item === 1)}
```

Comment:
The API is a bit strange, wouldn't is be more clear to use something like `{item === 1 && renderSecondaryItem()}`? Then you would be able to get rid of the confusing paramenter in the props.